### ISSUE TITLE
Request to Open the Discussions Section

### ISSUE THREAD

**Issue by foreverimagining** [Kaldek/rat-ratgdo#87]
I would like to request that the Discussions section be enabled for this repository. It probably is overdue, but we can at least streamline conversations going forward by enabling it now.

Reasons to use a Discussion section:
- **Community Gallery:** I think a pinned discussion that would be used for people to show off their boards would be beneficial. This gallery could lead to exposure to new ideas and interest in different functions that can lead to innovation and additions to the boards provided to the community. I have gotten my chimera RAT-RATGDO/ultrasonic-car-presence-sensor working with few changes in the device's yaml file, but before offering it up in a Pull Request, I would like to gauge community interest and gather opinions for a possible change in the implementation. I think this would be better (and more cleanly) done by using Discussions.
- **Cleaning up the Issues:** Many of the Issues have been closed (sometimes before the conversations have been resolved) because they're not really Issues or they would be better served as Pull Requests. There have also been a few Issues opened where people simply want to express gratitude, which, while very nice to see, isn't an issue to be dealt with. By opening Discussions, we can move these conversations over there and reserve the Issues section for problems that need to be addressed by contributors to the repository, whether that be original contributors or those who have added their own boards.
- **Dealing with Troubleshooting:** Occasionally, we have people who come to the repository who need help implementing the wealth of knowledge that has been accrued. It's a lot harder to come into the conversation later down the road and very easy to miss bits of information buried in a conversation from way back at the beginning that for others has become common knowledge. Instead of making an Issue for each case where simple troubleshooting is necessary to work through, it might be better to have a Discussion thread open where common issues are addressed in the first post (which would then be edited from time to time), and new problems not fixed by any of the standard troubleshooting techniques can be brought up in one localized conversation that will hopefully be easier to browse through for similar problems.
- **General Conversations:** Some conversations have appeared that could be categorized together, such as the other alternatives to the original RATGDO that have appeared on the market. The Discussions section would be perfect for such general, tangentially related conversations that interest the community. Then, if they develop in such a way that requires some kind of action in the repository, Issues or Pull Requests could be created relating and pointing to the conversations that preceded them.

TL;DR: The Discussions section would be good and useful.

**Comment by Kaldek** [comment]
Willdo, sorry for the delay.  My full time job decided to get in my way!
